be/src/common/config.h

```diff
@@ -536,6 +536,15 @@ namespace config {
     // Whether to continue to start be when load tablet from header failed.
     CONF_Bool(ignore_load_tablet_failure, "false");
 
+    // Set max cache's size of query results, the unit is M byte
+    CONF_Int32(cache_max_size, "256"); 
+
+    //Cache memory is pruened when reach cache_max_size + cache_elasticity_size
```

Comment:
cache memory will be shrinked?
i do not understand what does this mean....

Reply:
In order to avoid frequent cache cleaning and keep high hit rate, set two config item, cache_max_size  and cache_elasticity_size，such as default config value,  when reach 256M+128M，cache memory is pruned to 256M